Formula/pulumi.rb

```diff
@@ -0,0 +1,30 @@
+class Pulumi < Formula
+  desc "Define cloud applications and infrastructure in your favorite language"
+  homepage "https://pulumi.io/"
+  url "https://github.com/pulumi/pulumi.git",
+      :tag => "v0.15.0",
+      :revision => "23cbfa503d856f57a3e00599d53c04fe69c93cf0"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dir = buildpath/"src/github.com/pulumi/pulumi"
+    dir.install buildpath.children
+    (buildpath/"bin").mkpath
+
+    cd dir do
+      system "dep", "ensure", "-vendor-only"
+      system "make", "dist"
+      bin.install Dir["#{buildpath}/bin/*"]
```

Comment:
You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.

Reply:
I've added it but I don't really understand why I need things like README and LICENSE into the prefix.

Reply:
IANAL but I think most open-source licenses _at least_ imply an expectation of providing the license when you package that software in source or binary format. The formula `prefix` just happens to be where we put those.

Reply:
Makes sense, thanks.